x/bank/keeper/end_blocker.go

```diff
@@ -0,0 +1,13 @@
+package keeper
+
+import (
+	"context"
+)
+
+func (k BaseKeeper) EndBlocker(ctx context.Context) error {
+	if k.endBlockerCb != nil {
+		return k.endBlockerCb(&k, ctx)
```

Comment:
@vinhphuctadang let's make it generic and accessible to all modules by adding switch case